_maps/shuttles/shiptest/minutemen_carina.dmm

```diff
@@ -507,28 +621,25 @@
 /turf/open/floor/plasteel/dark,
 /area/ship/security/prison)
 "hs" = (
-/obj/structure/fans/tiny,
-/obj/machinery/door/poddoor{
-	id = "space_cops_bay";
-	name = "cargo bay blast door"
+/obj/structure/table/reinforced,
+/obj/item/radio/intercom/wideband{
+	dir = 8
 	},
-/obj/effect/decal/cleanable/dirt,
-/turf/open/floor/engine/hull,
-/area/ship/cargo)
-"hv" = (
-/obj/machinery/airalarm/all_access{
-	pixel_y = 24
+/obj/item/gps{
+	gpstag = "NTREC1";
```

Comment:
would change that to something more fitting to minutemen

Reply:
MMREC1

Reply:
I actually didn't even realize the GPS had a custom name. It was probably copied from some other ship at some point. I'll just reset the tag to its default value.